cfme/base/login.py

```diff
@@ -103,7 +103,6 @@ def logged_out(self):
 
     def logout(self):
         self.settings.select_item('Logout')
-        self.browser.handle_alert(wait=None)
```

Comment:
This needs to be here, I understand its causing the below exception, but removing this call is not the correct fix. Please see #8877  for the fix for this.

```
>           self.logger.info('handling alert: %r', popup.text)
E           AttributeError: 'NoneType' object has no attribute 'text'

cfme/utils/appliance/implementations/common.py:80: AttributeError
AttributeError
'NoneType' object has no attribute 'text'
```